### ISSUE TITLE
Can't make it turn on?

### ISSUE THREAD

**Issue by fribse** [slashback100/presence_simulation#52]
Hi @slashback100 
HA is core 2022.4
Presence simulation is 1.14

Hmm, I don't understand what's going on with this now.
I have the service and all showing up, but if I 'switch on' te simulation, it doesn't turn on?

My group set up in the integration looks like this:
```
presence_simulator:
  name: All entities for presence simulator
  entities:
    - light.all_commonareas
    - light.all_adrian
    - light.all_bedroom
    - light.all_office
    - cover.bedroom_blind
    - cover.livingroom_blind_big
    - cover.livingroom_small_window_blind
    - cover.office_window_blind
```

The 'all' lights are light groups.

**Comment by slashback100** [comment]
Do you have something in your log?

**Comment by fribse** [comment]
Only from the startup where some devices are unknown because they were
renamed?

tor. 7. apr. 2022 13.03 skrev slashback100 ***@***.***>:

**Comment by fribse** [comment]
Hmm, I got those sorted, and now I can activate it, I don't know if the recent update also did something?

**Comment by slashback100** [comment]
Yep, renaming a light shouldn't have an impact...

**Comment by slashback100** [comment]
Should be fixed by #57
